CONTRIBUTING.rst

```diff
@@ -64,22 +64,6 @@ repo, from which you can submit a pull request.
    $ cd ~/hack-on-gcloud
    $ $VENV/bin/python setup.py develop
 
-I'm getting weird errors... Can you help?
------------------------------------------
-
-Chances are you have some dependency problems...
-If you're on Ubuntu,
-try installing the pre-compiled packages::
-
-  $ sudo apt-get install python-crypto python-openssl libffi-dev
-
-or try installing the development packages
-(that have the header files included)
-and then ``pip install`` the dependencies again::
-
-  $ sudo apt-get install python-dev libssl-dev libffi-dev
```

Comment:
I think you may be overestimating. It's very common for people to not know what they're saying when native extensions don't build due to missing `Python.h`.